pymc/sampling/mcmc.py

```diff
@@ -925,9 +923,6 @@ def _mp_sample(
             strace = traces[error._chain]
             for strace in traces:
                 strace.close()
-
-            multitrace = MultiTrace(traces)
-            multitrace._report._log_summary()
```

Comment:
Here too: The `multitrace` can not have warnings that would be printed by `_log_summary()` because none were added here or in its `__init__`